src/main/java/step3/domain/Round.java

```diff
@@ -0,0 +1,20 @@
+package step3.domain;
+
+public class Round {
+
+    private int roundTime;
+    private int distance;
```

Comment:
이 두값은 외부에서 변경하거나, 설정된 이후에 변경되지 않는 값으로 보입니다.

```suggestion
    private final int roundTime;
    private final int distance;
```